frame/support/procedural/src/storage_alias.rs

```diff
@@ -549,6 +595,33 @@ fn generate_storage_instance(
 	let name = Ident::new(&format!("{}_Storage_Instance", storage_name), Span::call_site());
 	let storage_name_str = storage_name.to_string();
 
+	let mut counter_code = Option::None;
```

Comment:
```suggestion
```

Reply:
This is going to break the build, this variable needs to be top level.